pkg/cni/plugin/plugin.go

```diff
@@ -129,6 +133,63 @@ func enableXdpAuth(ifname string) error {
 	return nil
 }
 
+func enableTcEgress(args *skel.CmdArgs) error {
+	var (
+		err     error
+		link    netlink.Link
+		tc      *ebpf.Program
+		ifIndex int
+		output  bytes.Buffer
+	)
+	ifIndex = 0
+	ethtoolArgs := []string{"-S", args.IfName}
+
+	if tc, err = utils.GetProgramByName(constants.TC_MARK_ENCRYPT); err != nil {
+		return fmt.Errorf("failed to get tc program: %v", err)
```

Comment:
This is not right, we may not enable ipsec at all. We should not return eror when ipsec is disabled